src/tracker-component.js

```diff
@@ -218,6 +236,14 @@ export const Tracker = (config? : Config = {}) => {
             return event;
         },
         setUser: (data : UserData) => {
+            try {
+                validateUser(data);
```

Comment:
should be `data.user || data`

Reply:
![Screen Shot 2019-09-03 at 12 41 36 PM](https://user-images.githubusercontent.com/686933/64203447-47dace80-ce48-11e9-86c3-b3961aec9466.png)

left: v2. 
Right: v1

Reply:
<img width="389" alt="Screen Shot 2019-09-03 at 1 14 48 PM" src="https://user-images.githubusercontent.com/13542922/64205452-e10be400-ce4c-11e9-9c5f-008b7a4662b7.png">